make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -103,9 +102,7 @@ LIBAWT_EXTRA_HEADER_DIRS := \
     java.base:libjava \
     #
 
-LIBAWT_CFLAGS += -D__MEDIALIB_OLD_NAMES -D__USE_J2D_NAMES $(X_CFLAGS)
-
```

Comment:
Why is X_CFLAGS no longer needed ?

Reply:
It is still needed, and it is still there. It has just moved to inside the `SetupJdkLibrary` call.

The idea is that we should not really use additional variables if it is possible to give the proper values to the parameters directly in SetupJdkLibrary.

However, I see now that I have only partially succeeded with LIBAWT_CFLAGS. And also that I have handled CFLAGS and CXXFLAGS inconsistently. *sigh* I think I've been over this code a thousand times, and still I missed this.